jac-uk/apply
# #901 Anti-automation protection

## What's included?
Implement [Google reCAPTCHA v3](https://developers.google.com/recaptcha/docs/v3) during use of the login and forgotten password functionality in order to prevent the automated submission of such forms.

Note: this PR is related to [Verify recaptcha token](https://github.com/jac-uk/digital-platform/pull/752)

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Preview URL: https://jac-apply-develop--pr905-feature-901-anti-aut-jwo3kdy3.web.app

1. Go to `Sign In` page and check the functionality
2. Go to `Forgotten password` page and check the functionality

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

https://user-images.githubusercontent.com/79906532/184397238-9bdf3e39-5b8f-49e5-9384-5e990b7a78cd.mov

---
PREVIEW:DEVELOP
